common/daq/generation/gen_embedded_can.py

```diff
@@ -79,35 +79,79 @@ def gen_send_macro(lines, msg_config, peripheral):
     lines.append(f"    }} while(0)\n")
 
 def gen_filter_lines(lines, rx_msg_configs, peripheral):
+    # peripheral being passed in is node-wise
     """ generates hardware filters for a set of message definitions for a specific peripheral """
-    on_mask = False
-    filter_bank = 0
-    filter_bank_max = 27
-
-    if peripheral == "CAN1":
-        filter_bank = 0
-        filter_bank_max = 14
-    elif peripheral == "CAN2":
-        filter_bank = 14
-        filter_bank_max = 27
-    else:
-        print(f"Unknown CAN peripheral {peripheral}")
+    tlp = peripheral
+    on_mask_periph = {"CAN1": False, "CAN2": False} # separated for safety (i don't know what this does)
+    filter_bank_periph = {"CAN1": 0, "CAN2": 14}
+    filter_bank_max_periph = {"CAN1": 14, "CAN2": 27}
+
+    # these lines were default behavior, just moved into tuples above and symbols renamed
```

Comment:
Small thing, if you are talking about lines 85-87, those are dictionaries and not tuples.